client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -1626,6 +1669,23 @@ export default {
       }
     },
 
+    setStorageRecordListFilterPreset (filter = []) {
+      let currentListFilters = []
```

Comment:
Im thinking it might make sense to show a modal with input, so you can name your filter preset. Otherwise custom filter 1/2 and so on dont tell you much